packages/transaction-controller/src/utils/gas.ts

```diff
@@ -49,38 +55,44 @@ export async function estimateGas(
   const request = { ...txParams };
   const { data, value } = request;
 
-  const { gasLimit: gasLimitHex, number: blockNumber } = await getLatestBlock(
+  const { gasLimit: blockGasLimit, number: blockNumber } = await getLatestBlock(
     ethQuery,
   );
 
-  const gasLimitBN = hexToBN(gasLimitHex);
+  const blockGasLimitBN = hexToBN(blockGasLimit);
+
+  const fallback = BNToHex(
+    fractionBN(blockGasLimitBN, GAS_ESTIMATE_FALLBACK_BLOCK_PERCENT, 100),
+  );
 
   request.data = data ? add0x(data) : data;
-  request.gas = BNToHex(gasLimitBN.muln(GAS_ESTIMATE_FALLBACK_MULTIPLIER));
   request.value = value || '0x0';
 
-  let estimatedGas = request.gas;
-  let simulationFails;
+  delete request.gasPrice;
+  delete request.maxFeePerGas;
+  delete request.maxPriorityFeePerGas;
```

Comment:
nice fix! 

Q: could you please elaborate on these deleted props from the request?

Reply:
This matches the behaviour of the legacy extension controller, and ensures that some non-Infura chains don't validate the user balance and throw an error rather than returning a theoretical estimate.